packages/devtools_app/lib/src/screens/provider/instance_viewer/result.dart

```diff
@@ -23,35 +23,6 @@ class Result<T> with _$Result<T> {
   factory Result.data(T value) = _ResultData<T>;
   factory Result.error(Object error, StackTrace stackTrace) = _ResultError<T>;
 
-  factory Result.guard(T Function() cb) {
```

Comment:
please remove these changes since this file is generated

Reply:
Done